actionpack/test/controller/request/test_request_test.rb

```diff
@@ -6,6 +6,11 @@ def test_test_request_has_session_options_initialized
     assert @request.session_options
   end
 
+  def test_mutate_session_options
```

Comment:
we can make it `test_mutating_session_options_does_not_affect_default_options` :smile: